### PR TITLE
Integrate new AADL/AGREE to VDM to CSV translator toolchain

### PR DESCRIPTION
**Pull request by tuxji**
This request is safe to merge, but I'm giving you both the chance to 
see what changes I made.  After you rebuild and reimport the VERDICT
plugin into OSATE, I think things will work mostly as they did before, 
although I found a problem with the kind2 translator not being able
to parse its Lustre input (that happens when running CRV).  Here are
the changes I made:

In verdict-bundle-app's App.java, replace --aadl and --iml options
with --vdm option.  We will use --vdm only together with --crv at this
time.  The other toolchain options will continue receiving their
inputs from CSV files in the STEM project.  If --vdm <vdmFile> is
passed, derive the model's name (displayed in the console) from the
VDM file's base name without its .xml file suffix.  There were two
overloaded runMbas methods; remove the second runMbas method that was
being used with the --aadl/--iml options.  Update the runCrv method to
work with the --vdm option instead of the --aadl/--iml options and
remove the runAadl2impl method since it's no longer needed.  (We
haven't removed all other code having anything to do with aadl2iml
yet, but may do that later.)

In feature.xml, remove org.apache.common.math3 bundle since we don't
need it anymore.

In com.ge.research.osate.verdict.targetplatform.target, remove all
bundles (especially math3) being imported from the orbit update site.

In com.ge.research.osate.verdict.vdm/.classpath, correct the source
path for verdict-data-model.jar to point to the verdict-data-model
project in the workspace, not in someone's filesystem.

In com.ge.research.osate.verdict.vdm's MANIFEST.MF, remove unnecessary
Require-Bundle: org.apache.commons.math3; add lib/commons-math3.jar to
Bundle-ClassPath instead.

In com.ge.research.osate.verdict's MANIFEST.MF, remove unnecessary
Require-Bundle: org.apache.commons.math3; add all dependent jars
including lib/commons-math3.jar to Bundle-ClassPath instead (needed
slf4j-api and slf4j-simple too).

In com.ge.research.osate.verdict's pom.xml, add slf4j-simple
dependency just in case zt-exec tries to log something via slf4j-api
and use excludeGroupIds in copy-dependencies configuration to prevent
redundantly copying VERDICT jars into lib directory.

In CRVHandler.java, remove code needed to run aadl2iml, call
runAadl2Vdm instead, and pass VDM file to runBundle.  Add runAadl2Vdm
and make it use Aadl2Vdm and VdmTranlator to write VDM file to disk.
Modify runBundle to pass --vdm <vdmFile> instead of --aadl <args> to
verdict-bundle-app.

In GSNHandler.java, replace Aadl2CsvTranslator call with runAadl2Csv
call.  Make runAadl2Csv use Aadl2Vdm and Vdm2Csv to write CSV files to
STEM project.

In MBasHandler.java, replace Aadl2CsvTranslator call with runAadl2Csv
call.  Make runAadl2Csv use Aadl2Vdm and Vdm2Csv to write CSV files to
STEM project.

In MBasSynthesisHandler.java, replace Aadl2CsvTranslator call with
runAadl2CsvWithSynthesis call.  Make runAadl2CsvWithSynthesis use
Aadl2Vdm and Vdm2Csv to write CSV files to STEM project and VDM model
to verdict_model.xml (should we name the file the same as the model's
name instead?).

In Vdm2Csv.java, remove unused imports and obsolete comment.

### CODE REVIEW

**Comment by VidhyatvGE** [comment]
"_In CRVHandler.java, remove code needed to run aadl2iml, call
runAadl2Vdm instead, and pass VDM file to runBundle. Add runAadl2Vdm
and make it use Aadl2Vdm and VdmTranlator to write VDM file to disk.
Modify runBundle to pass --vdm instead of --aadl to
verdict-bundle-app._"  --> Aadl2Vdm translates only AADL constructs to VDM, we need call to Agree2Vdm.execute() to populate the AADL+AGREE data in VDM

**Comment by baoluomeng** [comment]
> "_In CRVHandler.java, remove code needed to run aadl2iml, call runAadl2Vdm instead, and pass VDM file to runBundle. Add runAadl2Vdm and make it use Aadl2Vdm and VdmTranlator to write VDM file to disk. Modify runBundle to pass --vdm instead of --aadl to verdict-bundle-app._" --> Aadl2Vdm translates only AADL constructs to VDM, we need call to Agree2Vdm.execute() to populate the AADL+AGREE data in VDM

Can you create two functions inside Aadl2Vdm for the translations of AADL+VERDICT to VDM and AADL+AGREE to VDM so that John only needs to invoke the call once?

File: tools/verdict/com.ge.research.osate.verdict/src/com/ge/research/osate/verdict/handlers/CRVHandler.java [hunk 1]
```diff
@@ -88,16 +86,25 @@ public void run() {
...
-	public static boolean runBundle(String bundleJar, String dockerImage, String inputPath, String outputPath,
-			String aadl2imlbin, String kind2bin, String propertySet) throws IOException {
+	/**
+	 * Calls Aadl2Vdm translator and writes model to vdmFile
+	 * @param dir
+	 * @param vdmFile
+	 */
+	public static void runAadl2Vdm(File dir, File vdmFile) {
+		Aadl2Vdm aadl2vdm = new Aadl2Vdm();
+		Model model = aadl2vdm.execute(dir);
```
Comment (VidhyatvGE) [inline tools/verdict/com.ge.research.osate.verdict/src/com/ge/research/osate/verdict/handlers/CRVHandler.java:96]: Need call to agree2vdm.execute here. Agree2vdm.execute first populates the vdm with aadl data by invoking aadl2vdm and then populates the AGREE data. However, Agree2vdm.execute has to be cleaned up to remove test code.

**Comment by tuxji** [comment]
OK, got it, we need to call agree2vdm.execute instead of aadl2vdm.execute everywhere that I was calling aadl2vdm.execute.  I'll let you merge and make the changes since I can't call agree2vdm.execute before you clean it up.

**Pull request merged**